fvtest/compilertriltest/JitTest.hpp

```diff
@@ -312,4 +313,111 @@ std::vector<std::tuple<L,R>> const_value_pairs()
 
 } // namespace CompTest
 
+/**
+ * @brief Enum values representing the reason for skipping a test
+ *
+ * These values are intended to be short descriptions of why a test is skipped
+ * and are mostly useful for logging and reporting purposes. Additional explenations
+ * should be specified in the skip message.
+ */
+enum SkipReason {
+   KnownBug,               // test is skipped because of a known bug
+   MissingImplementation,  // feature under test is not implemented
+   UnsupportedFeature,     // feature under test is not supported
+   NumSkipReasons_,        // DO NOT USE IN USER CODE
```

Comment:
Are we able to use `std::numeric_limits<enum SkipReason>::max()` or do not all our targets support it?

Reply:
I don't think that ever works: https://stackoverflow.com/questions/9201865/why-is-c-numeric-limitsenum-typemax-0 . :disappointed: